scripts/util.ts

```diff
@@ -0,0 +1,38 @@
+// Function to calculate Ratcliff-Obershelp string similarity
+// https://en.wikipedia.org/wiki/Gestalt_Pattern_Matching
```

Comment:
did you write this?

Reply:
Rog wrote it, I am merely documenting.

Reply:
Tis used in club and staff generation to match similar names because gunn cannot keep a consistent staff name across two (or the same, as a matter of fact) sites.

Reply:
glorious